workspaces/libnpmteam/lib/index.js

```diff
@@ -24,11 +24,11 @@ cmd.destroy = (entity, opts = {}) => {
   const { scope, team } = splitEntity(entity)
   validate('SSO', [scope, team, opts])
   const uri = `/-/team/${eu(scope)}/${eu(team)}`
-  return npmFetch.json(uri, {
+  return npmFetch(uri, {
     ...opts,
     method: 'DELETE',
     scope,
-  })
+  }).then((res) => res.text())
```

Comment:
Another way could be to have a `.text()` in the `npm-registry-fetch` lib and then use a `npmFetch.text()` here like `npmFetch.json()` is used.

Reply:
We don't have any responses that aren't json that we actually want to parse the results of, nor should we.